src/components/Link/link.css

```diff
@@ -0,0 +1,4 @@
+a.disabled {
```

Comment:
Do we need this? uikit should be handling it already. Please check. If it's not handling, please do the same thing using `styled-components` because using css files will cause scope issues.

Reply:
@theajr : Yes we need it, they are not handling it. As suggested removed css and added as style. Please review